File: Modul-4/Soal-2/PRAK402-2410817210021-MuhammadNafisPutra.c
#include <stdio.h>

int main() {
    int angka;

    scanf("%d", &angka);

    for (int i = 1; i <= angka; i += 2) {
        printf("%d ", i);
    }
    printf("\n");
    if (angka %2 !=0)
    angka--;
    
    for (int i = angka; i >= 2; i -= 2) {
        printf("%d ", i);
    }
    return 0;
}

File: Modul-4/Soal-2/PRAK402-2410817210021-MuhammadNafisPutra.py
angka = input()
angka = int(angka)

for i in range (1, angka+1, 2):
    print(i, end = " ")
    if (angka %2 !=0):
        angka -=1
print()            
   
for i in range (angka, 1, -2):
    print(i, end = " ")